Breakout-Room-Meetings/DevconSEA/UX.md

```diff
@@ -0,0 +1,190 @@
+# [CLS] Ethereum Magicians Infinite Endgames: UX 
+
+## [Ethereum Magicians Infinite Endgames Sessions 1](https://app.devcon.org/schedule/QRG8QW)
+
+## Summary: 
+UX has been at the forefront of Ethereum recently, as standards for Account and Chain Abstraction have been gaining significant traction in the space. Join us (literally! This panel will be “fishbowl style”) as we discuss the challenges that we will need to figure out first, such as cross-L2 key management, asset handling and transactions; avoiding fragmentation (liquidity, network, users); coordinating standards across L2s and wallets; and more
+
+## Video: https://www.youtube.com/watch?v=63w7kHh737w
+
+## Panelists: 
+Niha (Safe), Cody Crozier (Coinbase), Derek Chiang (ZeroDev), Mark Smargon (Fuse Network), Pedro Gomes (WalletConnect)
```

Comment:
If we are also writing down moderators, then I guess it should include me :)

In that case, it should say:

Moderator: Tom Teman (Ethereum Foundation)

(Or simply mention me as one of the panelists)